src/meta-srv/src/procedure/region_migration.rs

```diff
@@ -111,6 +112,8 @@ impl PersistentContext {
 pub struct Metrics {
     /// Elapsed time of downgrading region and upgrading region.
     operations_elapsed: Duration,
+    /// Elapsed time of flushing leader region.
+    flush_leader_region_elapsed: Duration,
```

Comment:
I don't think we need to include this in the migration operation timeout.

Reply:
Resolved in [f55902e](https://github.com/GreptimeTeam/greptimedb/pull/5995/commits/f55902e25aa608f25e2c35009dd694e204134036)